packages/amplify-graphql-api-construct-tests/src/__tests__/backends/migration/many-to-many/app.ts

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env node
```

Comment:
as above, is there an opportunity to use `packages/amplify-graphql-api-construct-tests/src/__tests__/backends/configurable-stack/app.ts` instead of a custom stack?